core/state/statedb.go

```diff
@@ -1150,6 +1239,22 @@ func (s *StateDB) handleDestruction(nodes *trienode.MergedNodeSet) (map[common.A
 	return incomplete, nil
 }
 
+// XXX check it's still needed
+// GetTrie returns the account trie.
+func (s *StateDB) GetTrie() Trie {
+	return s.trie
+}
```

Comment:
it's still needed, unless I can find a way to access the trie from other means in the transition and that `OpenStorageTrie` is no longer called in the case of verkle.